### ISSUE TITLE
Address accessibility issues (alt text) from AMP scan

### ISSUE THREAD

**Issue by jstrothman** [18F/18f.gsa.gov#3599]
## TTS Digital Council provided automated AMP scans and requires fixes by August 31st. 
Our site's issues are all related to alternate text for images: 38 images either have missing alt text, or it's uninformative such as a url for the text.
[AMP scan spreadsheet](https://docs.google.com/spreadsheets/d/19IgCzm3KxAy57TzkCmxRGitIvFhurI4p/edit#gid=1587429197)

## This issue is done when
- [ ] Alt attribute is either empty because the image is purely decorative, or has informative text

**Comment by adunkman** [comment]
Adding automated accessibility testing in CI will close #3121 and #3130 as well.
